pkg/sdk/go/sdk.go

```diff
@@ -347,7 +299,7 @@ func NewSDK(conf Config) SDK {
 	}
 }
 
-func (sdk mfSDK) sendRequest(req *http.Request, token, contentType string) (*http.Response, error) {
+func (sdk mfSDK) sendRequest(req *http.Request, token, contentType string) (*http.Response, errors.SDKError) {
```

Comment:
Internal methods should probably not return SDK errors. The point of SDK error is to encapsulate service HTTP response and extract error from it if there is any.